### ISSUE TITLE
Different web pages do not have different titles

### ISSUE THREAD

**Issue by flybot-nam-nguyenhoai** [skydread1/flybot.sg#249]
## Problem

The page title (shown in the browser tab bar) is always "Flybot" regardless of the page content. This hampers accessibility and SEO:
- Accessibility: Users cannot immediately determine what information the page contains, especially when switching tabs. Users that use assistive technologies are especially affected, since they rely on the page title for navigation (e.g., screen readers announce page titles aloud). This is considered non-compliance with WCAG Guideline 2.4.[^wcag]
- SEO: Page titles are considered an important ranking factor. The more descriptive and accurate the page title, the better.

### Suggestions

- Modify the page title when navigating to a new page, using JavaScript's `document.title` directly.
- Consider whether `react-helmet` can be used for cleaner DOM code.

[^wcag]: https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html
